cms/models.py

```diff
@@ -47,8 +81,94 @@ class HomePage(Page):
     ]
     parent_page_types = [Page]
     subpage_types = [
-        "CoursePage",
+        "CourseIndexPage",
         "ResourcePage",
+        "HomeProductPage",
+    ]
+
+    def _get_child_page_of_type(self, cls):
+        """Gets the first child page of the given type if it exists"""
+        child = self.get_children().type(cls).live().first()
+        return child.specific if child else None
+
+    @property
+    def product(self):
+        """
+        Gets the "HomeProductPage" section sub page.
+        """
+        return self._get_child_page_of_type(HomeProductPage)
+
+    def get_context(self, request, *args, **kwargs):
+        return {
+            **super().get_context(request),
+            "product_section_title": self.product.product_section_title
+            if self.product
+            else "Product Section Title",
```

Comment:
This should be removed. If the section title hasn't been set, we shouldn't show a title at all. We don't get any value from this default